Extensions/dnSpy.AsmEditor/Properties/dnSpy.AsmEditor.Resources.es-ES.resx

```diff
@@ -2527,4 +2530,10 @@
   <data name="CompilerDlgTabTitle" xml:space="preserve">
     <value>Compilador</value>
   </data>
+  <data name="ReplaceNopsInstrCommand" xml:space="preserve">
```

Comment:
These strings shouldn't be updated here, they get updated when I get the latest translations from https://crowdin.com/project/dnspy